include/seqan3/alphabet/nucleotide/nucl16.hpp

```diff
@@ -315,9 +333,140 @@ constexpr nucl16 nucl16::W{internal_type::W};
+// containers
+// ------------------------------------------------------------------
+
+namespace seqan3
+{
+    /*!\name Alphabet aliases
+     * \{
+     * \brief Other names (typedefs) for seqan3::nucl16
+     * \relates nucl16
+     */
+    using dna16 = nucl16;
+    using rna16 = nucl16;
+    using dna = nucl16;
+    using rna = nucl16;
+    //!\}
+
+} // namespace seqan3
+
+
+// ------------------------------------------------------------------
+// containers
+// ------------------------------------------------------------------
+
+namespace seqan3
+{
+
+//!\brief Alias for an std::vector of seqan3::nucl16.
+//!\relates nucl16
+using nucl16_vector = std::vector<nucl16>;
+
+
+/*!\brief Alias for an std::basic_string of seqan3::nucl16.
+ * \relates nucl16
+ *
+ * \attention
+ * Note that we recommend using @link nucl16_vector @endlink instead of nucl16_string in almost all situations.
+ * While the C++ style operations on the string are well supported, you should not access the internal c-string
+ * and should not use C-Style operations on it, e.g. the `char_traits::strlen` function will not return the
+ * correct length of the string (while the `.size()` returns the correct value).
+ */
+using nucl16_string = std::basic_string<nucl16, std::char_traits<nucl16>>;
+
+} // namespace seqan3
+
+// ------------------------------------------------------------------
+// literals
+// ------------------------------------------------------------------
+
+namespace seqan3::literal
+{
+
+/*!\brief nucl16 literal (returns @link nucl16_vector @endlink)
+ * \relates seqan3::nucl16
+ *
+ * You can use this string literal to easily assign to nucl16_vector:
+ *
+ *~~~~~~~~~~~~~~~{.cpp}
+ *     // these don't work:
+ *     // nucl16_vector foo{"ACGTTA"};
+ *     // nucl16_vector bar = "ACGTTA";
+ *
+ *     // but these do:
+ *     using namespace seqan3::literal;
+ *     nucl16_vector foo{"ACGTTA"_nucl16};
+ *     nucl16_vector bar = "ACGTTA"_nucl16;
+ *     auto bax = "ACGTTA"_nucl16;
+ *~~~~~~~~~~~~~~~
+ *
+ * \attention
+ * All user-defined literals are in the namespace seqan3::literal!
+ */
+
+inline nucl16_vector operator "" _nucl16(const char * s, std::size_t n)
+{
+    nucl16_vector r;
+    r.resize(n);
+
+    for (size_t i = 0; i < n; ++i)
+        r[i].assign_char(s[i]);
+
+    return r;
+}
+
+/*!\brief nucl16 string literal (returns @link nucl16_string @endlink)
+ * \relates seqan3::nucl16
+ *
+ * You can use this string literal to easily assign to nucl16_vector:
+ *
+ *~~~~~~~~~~~~~~~{.cpp}
+ *     // these don't work:
+ *     // nucl16_string foo{"ACGTTA"};
+ *     // nucl16_string bar = "ACGTTA";
+ *
+ *     // but these do:
+ *     using namespace seqan3::literal;
+ *     nucl16_string foo{"ACGTTA"_nucl16s};
+ *     nucl16_string bar = "ACGTTA"_nucl16s;
+ *     auto bax = "ACGTTA"_nucl16s;
+ *~~~~~~~~~~~~~~~
+ *
+ * Please note the limitations of seqan3::nucl16_string and consider using the @link operator""_nucl16 @endlink instead.
```

Comment:
_nucl16 returns to nucl16_vector
_nucl16s returns nucl16_string
Since we don't recommend nucl16_string, we also recommend _nucl16 instead of _nucl16s. :bulb: